token/client/src/token.rs

```diff
@@ -2618,7 +2647,10 @@ where
             .new_decryptable_available_balance(transfer_amount, source_aes_key)
             .map_err(|_| TokenError::AccountDecryption)?;
 
-        self.process_ixs(
+        // additional compute budget required for `VerifyTransferWithFee`
+        const TRANSFER_WITH_FEE_COMPUTE_BUDGET: u32 = 500_000;
```

Comment:
Sorry I didn't catch this on the first review, but what about the other version of transfer with fee: `confidential_transfer_transfer_with_fee_and_split_proofs` and the parallel version?

Reply:
Yeah, it seems like those don't seem to get caught on the CU limit. The split proofs (range proof, ciphertext validity proof, etc.) have smaller CUs since they are essentially chunks of the `VerifyTransferWithFee`. Some of those exceed 200k limit, but the CU averaging seems to allow them to pass.

The `VerifyTransferWithFee` is the culmination of all those split proofs and therefore, have huge CU requirement for which even the CU averaging is not sufficient to bring it down to 200k.

Reply:
I think it might be safe to allocate additional budget for those lighter instructions as well, but perhaps it could be done in a follow-up PR to unblock https://github.com/solana-labs/solana/pull/34695 before the feature freeze.

Reply:
Ah ok, as long as the other ones work, then all good!